swagger/v1/get_person_responses.yaml

```diff
@@ -0,0 +1,9 @@
+'406':
```

Comment:
Im not sure this is being used anywhere?

Reply:
This is intentional: it is raises a 406 (Not Acceptable) error if a client tries to use `GET /people/{person_id}` in version 1